README.md

```diff
@@ -66,22 +66,58 @@ String will allow you to get a string from arguments, such as `$ progname --stri
 var myString *string = parser.String("s", "string", ...)
 ```
 
+Selector works same as a string, except that it will only allow specific values.
+For example like this `$ progname --debug-level WARN`
+```go
+var mySelector *string = parser.Selector("d", "debug-level", []string{"INFO", "DEBUG", "WARN"}, ...)
+```
+
+StringList allows to collect multiple string values into the slice of strings by repeating same flag multiple times.
+Such as `$ progname --string hostname1 --string hostname2 -s hostname3`
+```go
+var myStringList *[]string = parser.StringList("s", "string", ...)
+```
+
+List allows to collect multiple values into the slice of strings by repeating same flag multiple times 
+(at fact - it is an Alias of StringList).
+Such as `$ progname --host hostname1 --host hostname2 -H hostname3`
+```go
+var myList *[]string = parser.List("H", "hostname", ...)
+```
+
 Flag will tell you if a simple flag was set on command line (true is set, false is not).
 For example `$ progname --force`
 ```go
 var myFlag *bool = parser.Flag("f", "force", ...)
 ```
 
-List allows to collect multiple values into the slice of strings by repeating same flag multiple times.
-Such as `$ progname --host hostname1 --host hostname2 -H hostname3`
+FlagCounter will tell you the number of times that  simple flag  was set on command line 
+(integer greater than or equal to 1 or 0 if not set).
+For example `$ progname -vv --verbose`
 ```go
-var myList *[]string = parser.List("H", "hostname", ...)
+var myFlagCounter *int = parser.FlagCounter("v", "--verbose", ...)
```

Comment:
there shouldn't be double dash in front of long name here

Reply:
done